prosody-plugins/token/util.lib.lua

```diff
@@ -23,15 +23,15 @@ local function _verify_token(token, appId, appSecret, roomName)
 
 	local issClaim = claims["iss"];
 	if issClaim == nil then
-		return nil, "Issuer field is missing";
+		return nil, "'iss' claim is missing";
```

Comment:
Claim is the particular entry/field like 'room', 'iss', so writing "part of the claim" is not accurate